src/Scripting/CSharpTest/ObjectFormatterTests.cs

```diff
@@ -13,14 +13,24 @@
 using Microsoft.CodeAnalysis.Scripting.Hosting.UnitTests;
 using ObjectFormatterFixtures;
 using Roslyn.Test.Utilities;
+using Roslyn.Utilities;
 using Xunit;
 
 namespace Microsoft.CodeAnalysis.CSharp.Scripting.Hosting.UnitTests
 {
+    extern alias Scripting;
+    using CoreClrShim = Scripting::Microsoft.CodeAnalysis.CoreClrShim;
```

Comment:
All using should be outside the namespace. Same with extern. That's the default we use.

Reply:
➡️ It is not possible to resolve this ambiguity with the standard code style for this project.

Reply:
How does moving the extern alias outside the namespace have any impact here?